web/site/js/vis/Node.js

```diff
@@ -287,15 +287,26 @@ export default class Node extends VObject {
             selbox.scale.set(selboxsize / NODE_SELECTION_SCALE, selboxsize / NODE_SELECTION_SCALE);
             selbox.anchor.set(0.5, 0.5);
             this.root.addChildAt(selbox, 0);
-            this._selbox = selbox
+            this._selbox = selbox;
+
+            const rangeCircleSize = this.radioRange;
+            let rangeCircle = new PIXI.Graphics();
+            rangeCircle.beginFill(0x20ff20, 0.2);
+            rangeCircle.drawCircle(0, 0, rangeCircleSize);
```

Comment:
nit: not sure if it would look better if the circle has a solid border line (e.g. using lineStyle).

Reply:
Good idea. Done.

Reply:
Have you got a picture of the final version?

Reply:
![image](https://user-images.githubusercontent.com/11165794/167235840-df1e0688-08a4-4f5d-9ba9-7721794af3d6.png)

Reply:
Looks great! Thanks.